src/sources/invictus.ts

```diff
@@ -50,13 +54,35 @@ export class StakedInvictusMarketSource implements MarketSource {
 
     const stakedInvictusPrice = invictusMarketData.price * invictusRatio;
 
+    const lockedStakedVault = await cache.query(
+      this.connection,
+      LOCKED_STAKED_INVICTUS_VAULT,
+      TokenAccountParser
+    );
+    const lockedStakedInvictusMint = await cache.queryMint(
+      this.connection,
+      LOCKED_STAKED_INVICTUS_MINT
+    );
+    const totalInvictusLockedStaked = lockedStakedVault.info.amount;
```

Comment:
Something I just realized in this function, this is an `any` type, and it is also possible that the vault queries in this function should not work if it's a non-mainnet connection. Not sure why we haven't seen these errors in the app, maybe can investigate in a separate PR.